docs/integrations/destinations/snowflake.md

```diff
@@ -287,8 +287,8 @@ Otherwise, make sure to grant the role the required permissions in the desired n
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                             |
 |:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
-| 0.4.57  | 2023-03-23 | [#23904](https://github.com/airbytehq/airbyte/pull/24405)  | Revert host validation change                                                                                                                       |
-| 0.4.56  | 2023-03-22 | [#23904](https://github.com/airbytehq/airbyte/pull/23904)  | Added host validation by pattern on UI                                                                                                              |
+| 0.4.58  | 2023-03-27 | [#24615](https://github.com/airbytehq/airbyte/pull/24615)  | Added host validation by pattern on UI                                                                                                              |
```

Comment:
minor nit: maybe instead of `added` to have this as `fixed` since the two changelog comments are the same between `0.4.56` and `0.4.48`

Reply:
good point, done